### ISSUE TITLE
CSC454/491 Collaboration

### ISSUE THREAD

**Issue by isghanim** [dcsil/CSC491#7]
**Describe the concern**
I'm a bit confused about how team collaboration will work on CSC491 assignments. In particular, [the second assignment](https://dcsil.github.io/CSC491/assignments/a2.html) is implying that we will be working with our CSC454 team members to develop the document. 

> Your second assignment will be to identify areas where your team has expertise and where it may lack expertise.
> You will be expected to work together and submit this assignment as a team.

**So my question is:** will our CSC454 members be expected to be also working on CSC491 assignments with us? Secondly, is this fair for them? To essentially be required to do the work of two courses despite not being enrolled in one?

**Why is this important?**
This is important because it affects what the distribution of work will be among our team. And people exclusively enrolled in CSC454 may not have been expecting to be involved in the CSC491 course work.

**Ideal solution**
I'm not sure what the ideal solution would be. I mostly just wanted to bring up the question, but perhaps CSC454 students should be able to choose their level of involvement in CSC491 assignments. However, assignments like Assignment 2 seem to implicitly require their involvement by the nature of the task.

**Comment by jules2689** [comment]
Hey @isghanim, Thank you so much for bringing this up. It's a great topic and I commend you on being open :sparkles:

You will be working in collaboration with your CSC454 counterparts. They don't need to be the ones who are writing though. You are more than welcome to write a bit about your team mates, ask them about their background, expertise, and experiences. I would recommend asking them/interviewing them and writing about your team. The CSC491 students will be responsible for the write up and, in their limited experience, try to identify where your team lacks insight/experience.

To summarize, by nature of the DCSIL program, CSC454 people should fully expect to answer questions to help you finish your work* and potentially work on the software you submit in CSC491. This is your joint startup after all.

Does this help answer?

* If they won't, let the teaching staff know. We're not monsters, we will work it out!

**Comment by isghanim** [comment]
Thank you @jules2689 for the helpful response! That makes sense to me, and it answers my question :)

I think it might be a good idea as well to communicate what you've explained to everyone, including CSC454 students, maybe as an announcement. Although of course I'll make sure to have this conversation within my own group as well.

Thanks again!

**Comment by jules2689** [comment]
I'll coordinate an announcement with Mario. Thanks for that suggestion.

Please close this issue if you feel it has been resolved ❤️✨

**Comment by isghanim** [comment]
Awesome, thank you!
